README.md

```diff
@@ -100,16 +105,16 @@ Sign your REST-assured request specification with a defined client credential:
 
 ```java
 given()
-    .baseUri("https://endpoint.net")
-    .filter(new RestAssuredEdgeGridFilter(credential))
+    .baseUri("https://akaa-baseurl-xxxxxxxxxxx-xxxxxxxxxxxxx.luna.akamaiapis.net")
```

Comment:
I would remove at all all baseUri for REST-assured, because we're providing it as a host in ClientCredential. Entering it twice is confusing for me and will be for an end-user.